lib/pages/upload/image_handler.dart

```diff
@@ -93,13 +93,13 @@ class _ImageHandlerState extends State<ImageHandler> {
                       _urlController.text; // this will act as our collected url
                   String _name = _nameController.text;
                   addUrl(_url, _name); // values are added to db here
-                  Scaffold.of(context)
-                      .showSnackBar(SnackBar(content: Text('Processing data')));
+                  ScaffoldMessenger.of(context).showSnackBar(
+                      const SnackBar(content: Text('Processing data')));
```

Comment:
??? I may just be paranoid but this really hurts me.

Reply:
it told me scaffold was deprecated and to use this instead